### ISSUE TITLE
Upgrade to AWS SDK for Ruby - Version 2

### ISSUE THREAD

**Issue by arelenglish** [refile/refile#150]
The gem "aws-sdk" is now on version 2 and has several breaking changes. If you bundle without specifying version 1.x, you'll get an uninitialized constant error because the AWS namespace is now Aws to allow using the old version and the new version of the gem side by side.

**Comment by jacobwgillespie** [comment]
In case anybody else is searching for this (like me), this results in errors like:

```
refile-0.5.3/lib/refile/backend/s3.rb:37:in `initialize': uninitialized constant Refile::Backend::S3::AWS (NameError)
```

and the temporary fix was:

```ruby
gem 'aws-sdk', '~> 1.61.0'
```

**Comment by jnicklas** [comment]
Just back from vacation. I'm planning on splitting out the AWS integration into a separate gem. I'll make sure we're compatible with v2 and have a proper dependency set up. Should get to this in the next couple of days.

**Comment by arelenglish** [comment]
:+1:

**Comment by IanJBriggs** [comment]
If your rails app depends on the AWS SDK gem. You can also use:

`gem 'aws-sdk', '~> 1.63.0'`

**Comment by JoshuaNovak919** [comment]
@jnicklas  Any progress on splitting out the AWS integration?

**Comment by lazaronixon** [comment]
It is solved? I am using aws-sdk 2.0.42 with no problems.

**Comment by lazaronixon** [comment]
#218
